actions.rb

```diff
@@ -0,0 +1,86 @@
+require './book'
+require './person'
+require './student'
+require './teacher'
+require './rental'
+
+module Actions
+  def list_books
+    @books.each { |book| puts book }
+  end
+
+  def list_people
+    @people.each { |person| puts person }
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    choice = gets.chomp
+    case choice
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'That is not a valid input'
+      nil
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    student = Student.new(age: age, name: name, parent_permission: parent_permission, classroom: @classroom)
```

Comment:
- [ ] This should have been placed into the `create_person` method and the input and input validation should have been handled by another class that is responsible for the user interactions. Remember the KISS and the `DRY` | `DOT (do one thing)` concepts. I'd suggest moving the input validation interaction to another class or the `main` class then move create and update operations to the create_person method for both student and teacher.

Reply:
@omar-labana, We just want to make sure we are crystal clear before submitting another pull request. Do you mean we should take the entire content of this method into the `create_person` method or just only lines 34 to 37? If we do so for the `create_student` method are we expected to take the `create_teacher` method to `create_person` file also since both `create_student` and `create_teacher` are all creating a person?

Reply:
@ntzwilly and @HENRYKC24 , take the creating into the `create_person` and validation `student` or `teacher` into another class.